frontend/pages/admin/IntegrationsPage/cards/ChangeManagement/ChangeManagement.tsx

```diff
@@ -43,7 +43,7 @@ const validate = (formData: IChangeManagementFormData) => {
     if (!repoURL) {
       errs.repository_url =
         "Git repository URL is required when GitOps mode is enabled";
-    } else if (!validUrl({ url: repoURL })) {
+    } else if (!validUrl({ url: repoURL, protocols: ["http", "https"] })) {
```

Comment:
Since we want the error message "Git repository URL must include protocol (e.g. https://)," it'd make sense to add a string check for "https://" | "http://" prefixes and set that error message for that specific case, and use this `validURL` check and below generic "must be valid" message otherwise